intelmq/bots/collectors/http/collector_http_stream.py

```diff
@@ -35,6 +38,8 @@ def init(self):
         self.set_request_parameters()
         self.session = create_request_session(self)
 
+        self.error_count = 0
```

Comment:
we could use a private `__error_count`, but that should not be a blocker

Reply:
Good idea